pandas/tests/sparse/test_pivot.py

```diff
@@ -45,10 +46,24 @@ def test_pivot_table(self):
         #                            values='E', aggfunc='sum')
         # tm.assert_frame_equal(res_sparse, res_dense)
 
-    def test_pivot_table_multi(self):
-        res_sparse = pd.pivot_table(self.sparse, index='A', columns='B',
-                                    values=['D', 'E'])
-        res_dense = pd.pivot_table(self.dense, index='A', columns='B',
-                                   values=['D', 'E'])
+    @pytest.mark.parametrize(
+        'func',
+        ['mean',
+         'std',
+         'var',
+         'sem',
+         pytest.param('median', marks=pytest.mark.xfail(
```

Comment:
Is this a know issue we have an issue open for already?

Reply:
no was discovered here, but I can't repro locally